test/e2e/common/node/container_probe.go

```diff
@@ -229,6 +229,25 @@ var _ = SIGDescribe("Probing container", func() {
 		RunLivenessTest(f, pod, 1, defaultObservationTimeout)
 	})
 
+	/*
+		Release: v1.20
+		Testname: Pod liveness probe, container exec timeout, restart
+		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call does not return within the timeout specified, liveness probe MUST restart the Pod. When ExecProbeTimeout feature gate is disabled, the timeout is ignored BUT a failing liveness probe MUST restart the Pod.
+	*/
+	ginkgo.It("should be restarted with a failing exec liveness probe that took longer than the timeout [NodeConformance]", func() {
```

Comment:
We can't add an e2e test directly to conformance suites. They need to run in CI for at least two weeks first.

Reply:
Got it. @chewong You want to split this off into a separate PR?

Reply:
we can just remove the [NodeConformance] tag from the spec

Reply:
O.K., removed the [NodeConformance] tag

Reply:
@ehashman lemme know removing `[NodeConformance]` was the right resolution here, thank you!